src/main/java/com/slcube/shop/common/exception/CustomExceptionHandler.java

```diff
@@ -12,4 +12,9 @@ public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
     protected ResponseEntity<CustomErrorResponse> handleCustomException(CustomException e) {
         return CustomErrorResponse.toResponseEntity(e.getErrorCode());
     }
+
+    @ExceptionHandler(value = {Exception.class})
+    protected ResponseEntity handleException(Exception e) {
+        return CustomErrorResponse.toResponseEntity(CustomErrorCode.INTERNAL_SERVER_ERROR);
+    }
```

Comment:
```suggestion
    @ExceptionHandler(Exception.class)
    protected ResponseEntity handleException(Exception e) {
        log.error("interval server error", e);
        return CustomErrorResponse.toResponseEntity(CustomErrorCode.INTERNAL_SERVER_ERROR);
    }
```


default exception인경우 logging 넣어주세요! log.error(~~) 로 해주면 나중에 의도치 않은장애가 나면 이거 보고 처리가 가능하기 때문입니다.